PicPi/KeychainService.swift

```diff
@@ -0,0 +1,76 @@
+//
```

Comment:
thoughts about using a cocoapod for this? like https://github.com/evgenyneu/keychain-swift (this is the first google search I found lol)

Reply:
Do you think we need that , or just stick to this simple file?
I usually use this SwiftKeychainWrapper for my projects.

Reply:
I'm not too familiar with storing/loading from Keychains is but is there support for storing array of things vs passing a Data object?